src/CMDParse.cpp

```diff
@@ -0,0 +1,57 @@
+#include "CMDParse.h"
```

Comment:
I'd suggest renaming the files to `ArgumentParser.h/cpp` for consistency with the class name.